classes/profile.php

```diff
@@ -201,4 +206,22 @@ public static function save(\ExcimerLog $log, int $reason, int $created, float $
             'flamedatad3' => $flamedatad3
         ]);
     }
+
+
+    /**
+     * Delete profiles created earlier than a given time.
+     *
+     * @authpr Nigel Chapman <nigelchapman@catalyst-au.net>
```

Comment:
Should be you?

Reply:
What about the exist delete_before_epoch_time function, should this be removed too or kept?

Reply:
Nigel did write this one. I merely pulled it accross.
I'm hanging on to the excimer_* classes for now. But they will eventually be removed entirely.